pkg/integrationtests/envutils_test.go

```diff
@@ -116,12 +116,18 @@ func newTestEnvironment() (*ControllerTestEnvironment, error) {
 
 	kubernetesClient, err := client.New(cfg, client.Options{Scheme: scheme.Scheme})
 	if err != nil {
-		kubernetesAPIServer.Stop()
+		err = kubernetesAPIServer.Stop()
+		if err != nil {
+			return nil, err
```

Comment:
You're ignore the error from `kubernetesClient.Create()`. You can add the two errors into the same error message.

Reply:
Done